src/components/CRow.vue

```diff
@@ -35,23 +64,52 @@ import api from '@/api';
 export default {
   name: 'CRow',
   props: {
+    priority: Number,
     message_id: String,
     name: String,
     args: Array,
-    kwargs: Object
+    kwargs: Object,
+    enqueued_datetime: String,
+    started_datetime: String,
+    end_datetime: String
   },
   data() {
     return {
       canCancel: true,
       onError: false,
       isCanceling: false,
       error: '',
+      nameState: this.name,
       txtBtnCancel: 'Cancel'
     };
   },
+  filters: {
+    elapsedTime(strTime) {
+      if (strTime == undefined) {
+        return '';
+      }
+      let intTime = Date.parse(strTime);
```

Comment:
could you add `'prefer-const': 'error'` to .eslintrc.js. It's always better to use const when you don't want to change a variable.

https://eslint.org/docs/rules/prefer-const